packages/docusaurus/src/server/plugins/__tests__/__snapshots__/init.test.ts.snap

```diff
@@ -27,3 +27,48 @@ Example valid plugin config:
 
 "
 `;
+
+exports[`sortConfig should sort route config correctly 1`] = `
+Array [
+  Object {
+    "component": "",
+    "path": "/community",
+  },
+  Object {
+    "component": "",
+    "path": "/some-page",
+  },
+  Object {
+    "component": "",
+    "path": "/",
```

Comment:
this is not in the correct place

Reply:
Actually this is intentional, sorting by root routes first makes it easier to sort by root routes. After all, it doesn't really matter where just root route is, as long as root route with subroutes is placed at the last. But I moved new sorting to the beginning as you asked, probably it is better for consistency as well.